rpcd/playbooks/ceph-mon.yml

```diff
@@ -0,0 +1,20 @@
+---
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Deploy mons
+  hosts: mons
```

Comment:
I think we should add 'max_fail_percentage: 0'. If the initial cluster can't get quorum the play will hang. Strictly speaking it could fail for one and the play should complete but I've seen lots of random stuff when one or more hosts fail part way through. Also I don't think there's any benefit it letting it complete for some of the hosts only to rerun it to address the ones that failed.

If you don't want to add this here I'll make a separate patch.